perf-tests/src/main/scala/sttp/tapir/perf/vertx/Vertx.scala

```diff
@@ -96,6 +96,9 @@ object VertxRunner {
 }
 
 object TapirServer extends ServerRunner { override def start = VertxRunner.runServer(Tapir.route(1)) }
-object TapirMultiServer extends ServerRunner { override def start = VertxRunner.runServer(Tapir.route(127)) }
+object TapirMultiServer extends ServerRunner { override def start = VertxRunner.runServer(Tapir.route(128)) }
+object TapirInterceptorMultiServer extends ServerRunner {
+  override def start = VertxRunner.runServer(Tapir.route(128, withServerLog = true))
+}
 object VanillaServer extends ServerRunner { override def start = VertxRunner.runServer(Vanilla.route(1)) }
-object VanillaMultiServer extends ServerRunner { override def start = VertxRunner.runServer(Vanilla.route(127)) }
```

Comment:
**issue:** This is strange, `SimpleGetMultiRoute` tests should have been failing with 127 endpoints. I need to re-run them and double check the results now.

Reply:
because you've been hitting endpoint 127 and they had numbers 0..126? :)

Reply:
Yeah :) But Gatling still logs throughput, even if it gets 404s ;)

Reply:
maybe we can expect 200 in the scenario?